OpenRA.Game/Manifest.cs

```diff
@@ -69,12 +69,13 @@ public class Manifest
 		public readonly string[] SoundFormats = { };
 		public readonly string[] SpriteFormats = { };
 		public readonly string[] PackageFormats = { };
+		public readonly string[] TeamColorPresets = { };
```

Comment:
Please define these on the `ColorValidator` instead of creating a new engine-level field.  `IGlobalModData` implementations can automatically parse information out of the mod.yaml: see `MapGrid` for a simple example.